hack/ci-deploy.sh

```diff
@@ -37,6 +37,12 @@ for f in $( find "${DEPLOY_DIR}"/ -type f -name '*.yaml' ); do
     sed -i -E -e "s~docker\.io/scylladb/scylla-operator:[^ @]+$~${OPERATOR_IMAGE_REF}~" "${f}"
 done
 
+# TODO: Replace it with ScyllaOperatorConfig field when available.
```

Comment:
Maybe we could make this a GitHub issue?

Reply:
We already had https://github.com/scylladb/scylla-operator/issues/836, but it was closed on Tomas' departure.
I created https://github.com/scylladb/scylla-operator/issues/2314, I'll add a ref in a comment.